main.tf

```diff
@@ -189,6 +189,17 @@ module "aurora_cluster" {
   }))
 }
 
+resource "aws_security_group_rule" "additional_ingress_rules" {
```

Comment:
Be sure you're adding support for the RDS instances (not aurora) as well please.

Reply:
Sure @tsaucier-sf. I am still working on the changes